src/components/DonutChart.js

```diff
@@ -242,16 +241,12 @@ const styles = {
     transform: 'translate(-50%, -50%)'
   },
   label: {
-    color: '#333',
-    fontSize: '14px',
-    fontWeight: '800'
-  },
-  pointer: {
-    cursor: 'pointer'
+    color: '#999',
```

Comment:
Can we use https://github.com/mxenabled/mx-react-components/blob/master/src/constants/Style.js#L11?